package.json

```diff
@@ -305,7 +305,7 @@
     "p-map": "^4.0.0",
     "p-retry": "^4.2.0",
     "papaparse": "^5.2.0",
-    "pdfmake": "^0.1.65",
+    "pdfmake": "git://github.com/bpampuch/pdfmake.git#66952b262d3f73e2827a34a1997e9692c0467687",
```

Comment:
NOTE: this is the unpublished v0.3.0-alpha

At the time of this PR, the latest stable release version is 0.2.4 (2021-11-10).

Reply:
Yeah, in the end, since we are using `yarn link` I don't think this change was necessary!